packages/browserify/src/index.js

```diff
@@ -232,7 +231,7 @@ function writeAutoPolicy(policy, configuration) {
     })
     fs.writeFileSync(
       configuration.policyPaths.debug,
-      jsonStringify(policy, { space: 2 })
+      jsonStringifySortedPolicy(policy)
```

Comment:
why we no longer specifying space 2? or is it the `lavamoat-core` default

Reply:
I think it now happens within this new sink function

Reply:
It's the json-stable-stringify settings, which I'm providing just once, where it's now used